outputs.tf

```diff
@@ -0,0 +1,65 @@
+################################################################################
+# mongodbatlas_project
+################################################################################
+
+output "project_id" {
+  value = mongodbatlas_project.project.id
+}
+
+output "creation_timestamp" {
+  value = mongodbatlas_project.project.created
+}
+
+################################################################################
+# mongodbatlas_project_ip_access_list
+################################################################################
+
+output "ips_id" {
+  value = [for i in mongodbatlas_project_ip_access_list.ip :
+    {
+      ip_id = i.id
+    }
```

Comment:
would it be worth including `ip_address` as well here? Just to make it simple to see what each id corresponds to.

Reply:
We could add it, indeed. I was thinking more that the user can retrieve the id, and then once has the id, can then retrieve the ip_address if wanted. But it would be good as well to just retrieve the id along with its ip_address directly.